pkg/occlient/occlient.go

```diff
@@ -102,6 +102,9 @@ const (
 	// timeout for getting the default service account
 	getDefaultServiceAccTimeout = 1 * time.Minute
 
+	// timeout for waiting for project deletion
+	waitForProjectDeletionTimeOut = 3 * time.Minute
```

Comment:
Can user modify this timeout by updating odo preferences? If no, should we allow to do it?

cc @kadel @girishramnani

Reply:
Heck, you're right. We can add this.

Reply:
Opened a user story https://github.com/openshift/odo/issues/3199.